Client/Frontend/Browser/BrowserViewController/BrowserViewController+ProductNotification.swift

```diff
@@ -0,0 +1,161 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import BraveShared
+import BraveUI
+import Shared
+
+// MARK: - ProductNotification
+
+extension BrowserViewController {
+    
+    // MARK: Internal
+    
+    @objc func presentEducationalProductNotifications() {
+        // Adding slight delay so the popover notification will be shown after page loaded
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1) { [weak self] in
```

Comment:
Can we leave all code logic on main thread and add delay only to `notify..` methods? maybe even place this delay in `showBenchmarkNotificationPopover` method instead

Reply:
There are 2 reasons for this delay thats why it is used here however action method is separated from functionalty and comments are added to explain the reasons